docs/commands-details/workload_create_update_apply.md

```diff
@@ -402,17 +407,17 @@ tanzu apps workload apply tanzu-java-web-app --git-repo https://github.com/vmwar
 
 ### <a id="apply-git-tag"></a> `--git-tag`
 
-The tag in a Git repository from which the workload is created. This is used with `--git-commit`
-or `--git-branch`.
+The tag in a Git repository from which the workload is created. Can be unset by defining it as empty string when applying a workload (`--git-tag ""`).
 
 ### <a id="apply-git-commit"></a> `--git-commit`
 
-Commit in Git repository from where the workload is resolved. Can be used with `--git-branch` or `git-tag`.
+Commit in Git repository from where the workload is resolved. Either `--git-branch` or `--git-tag` can be specified with it too.
+It can be unset by defining it as empty string when applying a workload (`--git-commit ""`).
 
 <details><summary>Example</summary>
 
 ```bash
-tanzu apps workload apply tanzu-java-web-app --git-repo https://github.com/vmware-tanzu/application-accelerator-samples --sub-path tanzu-java-web-app --git-tag tap-1.3 --git-commit 1c4cf82e499f7e46da182922d4097908d4817320 --type web
+tanzu apps workload apply tanzu-java-web-app --git-repo https://github.com/vmware-tanzu/application-accelerator-samples --sub-path tanzu-java-web-app --git-commit 1c4cf82e499f7e46da182922d4097908d4817320 --type web
```

Comment:
--git-tag  needed here for this example?

Reply:
No, the idea is to let users know they do not need to provide tag or branch when using a commit. It was a bug that we had previously.